src/Text/Pandoc/Extensions.hs

```diff
@@ -118,6 +118,7 @@ data Extension =
     | Ext_literate_haskell    -- ^ Enable literate Haskell conventions
     | Ext_markdown_attribute      -- ^ Interpret text inside HTML as markdown iff
                                   --   container has attribute 'markdown'
+    | Ext_markdown_github_wikilink -- ^ Interpret a markdown wiki link
```

Comment:
I think I'd favor the name `ext_wikilink`.  This is a pretty generic wikilink format, certainly not GitHub specific.

Reply:
apparently it's inspired by mediawiki.... and [apparently](https://help.github.com/en/github/building-a-strong-community/editing-wiki-content) you can also use mediawiki image syntax:

    [[https://github.com/username/repository/blob/master/img/octocat.png|alt=octocat]]

but yeah, in markdown the link and image syntaxes are also closely related, so why not `ext_wikilink` :)